routing.go

```diff
@@ -0,0 +1,18 @@
+package veneur
+
+import "time"
+
+type ComputationRoutingConfig struct {
+	Name                    string         `yaml:"name"`
+	FlushGroup              string         `yaml:"flush_group"`
+	MatcherConfigs          MatcherConfigs `yaml:"match"`
+	WorkerInterval          time.Duration  `yaml:"worker_interval"`
+	WorkerWatchdogIntervals int            `yaml:"worker_watchdog_intervals"`
+	WorkerCount             int            `yaml:"worker_count"`
+	ForwardMetrics          bool           `yaml:"forward_metrics"`
+}
+
+type SinkRoutingConfig struct {
+	Name                    string   `yaml:"name"`
+	FlushGroupSubscriptions []string `yaml:"flush_group_subscriptions"`
```

Comment:
We moved `MatcherConfigs` to `ComputationRoutingConfig` because the tradeoff of additional expressivity at the sink level didn't seem worth the additional complexity it introduced to flusher.go. If you don't want to flush metrics, why not just not process them in the first place?

Reply:
Per conversation with @arnavdugar-stripe, there is likely future utility in matching at the sink level. You could imagine a change, for example, where computation routing is expanded to include dimension renaming a la `relabel_config`, where a timeseries would be different at ingest than at flush. (This already exists in the form of tags exclusion, common dimensions, etc)